build/edge/Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY . /go/src/github.com/kubeedge/kubeedge
 RUN apk --no-cache update && \
 apk --no-cache upgrade && \
 apk --no-cache add build-base linux-headers sqlite-dev binutils-gold && \
-CGO_ENABLED=1 GO111MODULE=off go build -v -o /usr/local/bin/edgecore -ldflags="${GO_LDFLAGS} -w -s -extldflags -static" \
+CGO_ENABLED=0 GO111MODULE=off go build -v -o /usr/local/bin/edgecore -ldflags="${GO_LDFLAGS} -w -s -extldflags -static" \
```

Comment:
If I use the `-static` parameter instead of `CGO`, will the `edgecore's` sqlite work? Have you tested it?

Reply:
no, so i open this PR to trigger CI, and i'm curious about why test is OK?